src/controllers/userProfileController.js

```diff
@@ -706,7 +706,7 @@ const userProfileController = function (UserProfile) {
     }
     // Check if the requestor has the permission to update passwords.
     const hasUpdatePasswordPermission = await hasPermission(
-      requestor.role,
+      requestor,
```

Comment:
The bug fix looks great.

But it seems the main effective change in this PR is this line, with other modifications likely introduced by your code formatter.

I recommend reverting these automatic formatting changes to maintain our established coding style.

Reply:
Thanks for noticing this! I have made another PR[766](https://github.com/OneCommunityGlobal/HGNRest/pull/766#issue-2151601903) to fix the lint.